pal/linux/target/rpi3/pal_ifx_i2c_config.c

```diff
@@ -43,8 +43,8 @@
 
 pal_linux_t linux_events = {0};
 
-gpio_pin_t gpio_pin_vdd = 27;
-gpio_pin_t gpio_pin_reset = 17;
+#define GPIO_PIN_VDD 27
+#define GPIO_PIN_RESET 4
```

Comment:
We have this guidance https://github.com/infineon/personalize-optiga-trust/blob/master/using-amazon-root-ca/step-2-setup-preparation.md#hardware-required-2
The figure is used not only there. Do you see advantages of having pin number 4 instead of 17?

Reply:
Sorry, the pin change was not meant to go into the commit, these pins are used for an internal board with the Trust X on it.